docs/docs/coverage/language/nodejs.md

```diff
@@ -55,6 +55,9 @@ By default, Trivy doesn't report development dependencies. Use the `--include-de
 ### pnpm
 Trivy parses `pnpm-lock.yaml`, then finds production dependencies and builds a [tree][dependency-graph] of dependencies with vulnerabilities.
 
+!!! warning
```

Comment:
I think this is just information for users.

```suggestion
!!! info
```

Reply:
You are right. `warning` looks too much.
Changed in https://github.com/aquasecurity/trivy/pull/6510/commits/924c8599aee77c6cb7895220d6ea655ea61d1651